### DISCUSSION TITLE
Configuration Docs

### DISCUSSION THREAD

**Discussion by stevefxp1** [Security-Onion-Solutions/securityonion#5231] [Unsupported Versions] [answered]
Are there any config docs on how to setup Security Onion? I would like to deploy an inline appliance. right behind my firewall, so that all traffic in/out has to pass through the Security Onion appliance. I do not want to sniff traffic. I cannot find anything that describes this type of setup.

**Comment by dougburks** [comment, marked as answer]
As mentioned in [your other discussion](https://github.com/Security-Onion-Solutions/securityonion/discussions/5198), we don't support running inline:
https://docs.securityonion.net/en/2.3/faq.html#can-security-onion-run-in-ips-mode

You can find all of our documentation here:
https://docs.securityonion.net/en/2.3/
